src/types/service.ts

```diff
@@ -1,5 +1,6 @@
-import * as Database from "@db/types";
-import * as Config from "@config/types";
+import { Data } from "./data";
+import { Execution } from "./execution";
+import * as Config from "./config";
```

Comment:
@nikonikoniko maybe just use `import { PartialConfig} from "./config"` since it's the only import that we use?

Reply:
> @nikonikoniko Looks good to me :) Was just wondering whether we'd also move types for the contexts?

@tlmn do you want to do that?  I didn't want to do so much digging :)

Reply:
> @nikonikoniko maybe just use `import { PartialConfig} from "./config"` since it's the only import that we use?

sure, sounds good

Reply:
@nikonikoniko I commited and pushed both changes